srcpkgs/golang-migrate/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'golang-migrate'
+pkgname=golang-migrate
+version=4.17.1
+revision=1
+build_style=go
+go_import_path="github.com/golang-migrate/migrate/v4"
+short_desc="Database migrations CLI written in Golang"
+maintainer="Thomas Treffry <t.treffry@gmail.com>"
+license="MIT"
+homepage="https://github.com/golang-migrate/migrate"
+distfiles="https://github.com/golang-migrate/migrate/archive/refs/tags/v${version}.tar.gz"
+checksum="59a92e27d58f22ce3270086d385b3548a2a44e09f91b0eda3e245f94bcf8f081"
+
+do_build() {
+	VERSION="$version" make build
```

Comment:
don't use the makefile, emulate it with these variables: https://github.com/void-linux/void-packages/blob/master/Manual.md#go-packages

Reply:
Thanks for the feedback! I have refactored the template to make use of the go package options